tsconfig.json

```diff
@@ -12,10 +12,10 @@
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
     "strictNullChecks": true,
     "suppressImplicitAnyIndexErrors": true,
-    "noUnusedLocals": true
+    "noUnusedLocals": false
```

Comment:
Why this one? It doesn't seem too strict, is it?

Reply:
Otherwise you can't compile while you are developing.